@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/pfz/fn_pfzReceive.sqf

```diff
@@ -0,0 +1,24 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_pfzReceive
+
+Description:
+    sends out the target data
```

Comment:
You have the send and receive descriptions the wrong way round. I would also recommend being more specific on what it is doing - broadcasting out the currently selected PFZ for other helis or receiving that PFZ